diff --git a/components/resources/BUILD.gn b/components/resources/BUILD.gn
index d1645a13a358b64cefb80cfe1c8f0691a68e3204..8e07af1390e9d1a5999aa543e6537f5daa8b096b 100644
--- a/components/resources/BUILD.gn
+++ b/components/resources/BUILD.gn
@@ -73,6 +73,7 @@ grit("components_scaled_resources") {
 }
 
 action("about_credits") {
+  deps = [ "//brave/components/resources:about_credits" ]
   script = "//tools/licenses.py"
   depfile = "$target_gen_dir/$target_name.d"
 
